wolfcrypt/src/ecc.c

```diff
@@ -7950,7 +7950,23 @@ int wc_ecc_import_x963_ex(const byte* in, word32 inLen, ecc_key* key,
 #ifdef WOLFSSL_SILABS_SE_ACCEL
     err = silabs_ecc_import(key, keysize);
 #endif
+#ifdef WOLFSSL_CRYPTOCELL
+    const CRYS_ECPKI_Domain_t* pDomain;
```

Comment:
Variable declarations should be at top of function or in brace section.

Reply:
Done. Txs